drone-webdriver/src/main/java/org/jboss/arquillian/drone/webdriver/factory/ChromeDriverFactory.java

```diff
@@ -103,6 +103,12 @@ public Capabilities getCapabilities(WebDriverConfiguration configuration, boolea
         new ChromeDriverBinaryHandler(capabilities).checkAndSetBinary(performValidations);
 
         ChromeOptions chromeOptions = new ChromeOptions();
+
+        String browser = configuration.getBrowser();
+        if (browser.equals("chromeHeadless") || browser.equals("chromeheadless")) {
```

Comment:
You are completely right. What I meant by my comment is that setting ChromeOptions can be done also for `chrome` - no only `chromeHeadless`. Of course, part of the logic will be check if the user has used `chromeHeadless` as a browser or not - if yes then set the appropriate argument there.
Setting ChromeOptions in the case of RemoteWebdriver was obviously forgotten.